core/src/main/java/nu/yona/server/analysis/service/DayActivityDto.java

```diff
@@ -78,7 +80,14 @@ public int getTotalMinutesBeyondGoal()
 
 	public static LocalDate parseDate(String iso8601)
 	{
-		return LocalDate.parse(iso8601, ISO8601_DAY_FORMATTER);
+		try
+		{
+			return LocalDate.parse(iso8601, ISO8601_DAY_FORMATTER);
```

Comment:
Wouldn't it be good to apply the same pattern at `WeekActivityDto.parseDate`?

Reply:
Certainly! Added it.